qt/applications/workbench/workbench/plotting/config.py

```diff
@@ -39,6 +39,10 @@ def initialize_matplotlib():
     mpl.rcParams["figure.dpi"] = QApplication.instance().desktop().physicalDpiX()
     # Hide warning made by matplotlib before checking our backend.
     warnings.filterwarnings("ignore", message="Starting a Matplotlib GUI outside of the main thread will likely fail.")
+    # Disabling default key shortcuts for toggling axes scale
+    mpl.rcParams["keymap.xscale"].remove("k")
+    mpl.rcParams["keymap.xscale"].remove("L")
+    mpl.rcParams["keymap.yscale"].remove("l")
```

Comment:
This disables the default keymaps, I am not sure if this is the best place to put it?

Reply:
do you need to remove upper case L?

Reply:
I think so, because `L` does the same thing as `k` in the original keymaps. So if I leave it in, `k` will be handled by us but `L` will default to the previous flakey behaviour

Reply:
okay 👍 and just to check `K` doesn't do the same thing as `l`?

Reply:
Yeah I just ckecked, `K` doesn't do anything :+1: